src/index.ts

```diff
@@ -75,6 +201,70 @@ export default class BlueButton {
         config.environment === Environments.PRODUCTION
           ? PRODUCTION_BASE_URL
           : SANDBOX_BASE_URL,
+      pkce: config.pkce ? config.pkce : true,
+      retryConfig: config.retryConfig,
     };
   }
+
+  async getResource(fhirReq: FhirRequest) {
```

Comment:
Why create this function separate? Why not make it part of the `FhirRequest` class? Or potentially change `FhirRequest` into a function instead?

Reply:
it seems that async function is still a global function (there is no 'this') even when it is put in a class.